src/platform_impl/macos/view.rs

```diff
@@ -357,13 +357,10 @@ extern "C" fn reset_cursor_rects(this: &Object, _sel: Sel) {
     }
 }
 
-extern "C" fn has_marked_text(this: &Object, _sel: Sel) -> BOOL {
-    unsafe {
-        trace!("Triggered `hasMarkedText`");
-        let marked_text: id = *this.get_ivar("markedText");
-        trace!("Completed `hasMarkedText`");
-        (marked_text.length() > 0) as i8
-    }
+extern "C" fn has_marked_text(_this: &Object, _sel: Sel) -> BOOL {
+    trace!("Triggered `hasMarkedText`");
+    trace!("Completed `hasMarkedText`");
```

Comment:
`hasMarkedText` is no longer getting triggered, so these logging calls shouldn't be necessary